content/en/docs/languages/java/configuration.md

```diff
@@ -0,0 +1,413 @@
+
+The autoconfigure module registers Java shutdown hooks to shut down the SDK when
+appropriate. Please note that since this project uses `java.util.logging` for
+all of it's logging, some of that logging may be suppressed during shutdown
+hooks. This is a bug in the JDK itself, and not something we can control. If you
+require logging during shutdown hooks, please consider using `System.out` rather
+than a logging framework that might shut itself down in a shutdown hook, thus
+suppressing your log messages. See this
+[JDK bug](https://bugs.openjdk.java.net/browse/JDK-8161253) for more details.
+
+{{% alert title="Signal configuration" color="info" %}}
+
+The text placeholder `{signal}` refers to the supported
+[OpenTelemetry Signal](/docs/concepts/signals/). Valid values include `traces`,
+`metrics`, and `logs`.
+
+{{% /alert %}}
+
+### Configuration Priority
+
+Signal specific configurations take priority over the generic versions.
+
+For example, if you set both `otel.exporter.otlp.endpoint` and
+`otel.exporter.otlp.traces.endpoint`, the latter will take precedence.
+
+### Disabling OpenTelemetrySdk
+
+The OpenTelemetry SDK can be disabled entirely. If disabled,
+`AutoConfiguredOpenTelemetrySdk#getOpenTelemetrySdk()` will return a minimally
+configured instance (i.e. `OpenTelemetrySdk.builder().build()`).
+
+| System property     | Description                               | Default |
+| ------------------- | ----------------------------------------- | ------- |
+| `otel.sdk.disabled` | If `true`, disable the OpenTelemetry SDK. | `false` |
+
+### Exporters
+
+Exporters output the telemetry. The following configuration properties are
+common to all exporters:
+
+| System property                               | Purpose                                                                                                                                                                                               |
+| --------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `otel.{signal}.exporter`                      | List of exporters to be used for {signal}, separated by commas. Default is `otlp`. `none` means no auto-configured exporter.                                                                          |
+| `otel.java.experimental.exporter.memory_mode` | If `reusable_data`, enable reusable memory mode (on exporters which support it) to reduce allocations. Default is `immutable_data`. This option is experimental and subject to change or removal.[^1] |
+
+[^1]:
+
+Exporters which adhere to
+`otel.java.experimental.exporter.memory_mode=reusable_data` are
+`OtlpGrpcMetricExporter`, `OtlpHttpMetricExporter`, and `PrometheusHttpServer`.
+Support for additional exporters may be added in the future.
+
+#### OTLP exporter (span, metric, and log exporters)
```

Comment:
We should mention the artifact coordinates in each place where we describe configuration that requires a particular artifact to be included in order to work.

I think this comes up for each of the exporter sections (otlp, logging, logging-otlp, zipkin, prometheus), and for the propagators.